Makefile

```diff
@@ -18,6 +18,39 @@ kube_informers_package   = $(kube_base_output_package)/client/informers
 path_apis                = "./core/..."
 header_file              = "header.go.txt"
 
+# protobuf
+out_path = /tmp
+
+gogofast_plugin_prefix := --gogofast_out=plugins=grpc,
+
+comma := ,
+empty :=
+space := $(empty) $(empty)
+
+importmaps := \
+	gogoproto/gogo.proto=github.com/gogo/protobuf/gogoproto \
+	google/protobuf/any.proto=github.com/gogo/protobuf/types \
+	google/protobuf/descriptor.proto=github.com/gogo/protobuf/protoc-gen-gogo/descriptor \
+	google/protobuf/duration.proto=github.com/gogo/protobuf/types \
+	google/protobuf/struct.proto=github.com/gogo/protobuf/types \
+	google/protobuf/timestamp.proto=github.com/gogo/protobuf/types \
+	google/protobuf/wrappers.proto=github.com/gogo/protobuf/types \
+	google/rpc/status.proto=istio.io/gogo-genproto/googleapis/google/rpc \
+	google/rpc/code.proto=istio.io/gogo-genproto/googleapis/google/rpc \
+	google/rpc/error_details.proto=istio.io/gogo-genproto/googleapis/google/rpc \
+	google/api/field_behavior.proto=istio.io/gogo-genproto/googleapis/google/api \
+
+mapping_with_spaces := $(foreach map,$(importmaps),M$(map),)
+gogo_mapping := $(subst $(space),$(empty),$(mapping_with_spaces))
+
+gogofast_plugin := $(gogofast_plugin_prefix)$(gogo_mapping):$(out_path)
+
+protoc = protoc -I. ${gogofast_plugin} --deepcopy_out=${out_path}
```

Comment:
Is it possible to run protoc from vendor like we are doing with the others? This way it doesn't depend on the right version to be on the container/user env.

Reply:
nope, it's written in C++... we can use bazel (which is how they do it in their examples) but I'd rather have it in the build container

Reply:
could we use k8s protogen to generated the .proto files from a go type?

Reply:
no- these are not k8s types, it's a gRPC service definition